src/abstract/JB721Hook.sol

```diff
@@ -182,6 +183,7 @@ abstract contract JB721Hook is ERC721, IJB721Hook, IJBRulesetDataHook, IJBPayHoo
     /// @param directory A directory of terminals and controllers for projects.
     constructor(IJBDirectory directory) {
         DIRECTORY = directory;
+        METADATA_ID_TARGET = address(this);
```

Comment:
Do we need this? getId(string) is doing the same thing no @mejango ?

Reply:
address(this) is storing the original address that each clone will then use as its metadata Id target